packages/kbn-securitysolution-autocomplete/src/field/index.tsx

```diff
@@ -1,17 +1,23 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
```

Comment:
Nit: personally I'm not a fan of folder-per-thing structure where inside of each folder you have a single `index.ts` or something like that. Makes it harder to use inside VS Code:

![](https://puu.sh/HYhq6/54d709b0f7.png)

I'd lean towards less folders and more uniquely named files.

Reply:
I think from this comment and the comment below, maybe we can do a follow up code sweep or discussion around file name changes, conventions?

I have been following mostly patterns seen within these folders to make things portable if we collapse between this package and the others, but I'm open if we want to do a broad cleanup/sweep of just naming changes in 1 faster PR.